src/components/Views/CameraMainView/CameraMainView.js

```diff
@@ -107,8 +130,8 @@ class CameraMainView extends React.Component<
         this.setState({ loading: true });
 
         const data = await this.camera.takePictureAsync(options);
-
         this.setState({ loading: false });
+        CameraRoll.saveToCameraRoll(data.uri);
```

Comment:
Hmmm we might want to put this in a util function called `saveToCameraRoll` that requests permissions if not granted, saves to camera roll, and deletes the photo from app cache. This would prevent dupes